test/pdfs/issue10572.pdf.link

```diff
@@ -0,0 +1,2 @@
+https://github.com/mozilla/pdf.js/files/2886648/bug.pdf
```

Comment:
Yeah, didn't add these because I thought that might be the case and asked that over on a different merge request. I've taken the originals and simplified them a bit with qpdf to remove all the fonts and text drawing. And then added them to the test_manifest.